apps/signing/tests/test_views.py

```diff
@@ -124,6 +126,35 @@ def test_version_already_uploaded(self):
         assert response.status_code == 409
         assert response.data['error'] == 'Version already exists.'
 
+    @mock.patch('devhub.views.auto_sign_version')
+    def test_version_added_is_experiment(self, sign_version):
+        self.grant_permission(self.user, 'Experiments:submit')
+        guid = 'experiment@xpi'
+        qs = Addon.unfiltered.filter(guid=guid)
+        assert not qs.exists()
+        response = self.put(
+            addon=guid, version='0.1',
+            filename=os.path.join('apps/files/fixtures/files/experiment.xpi'))
```

Comment:
I think you can leave out the `os.path.join` here, I don't think it's doing anything.